src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -260,30 +270,55 @@ function positionTooltip(chartType: any, stacking: any, labelWidth: any, labelHe
     };
 }
 
-const showFullscreenTooltip = () => {
+export function getTooltipPositionInViewPort(
+    chartType: string,
+    stacking: boolean,
+    labelWidth: number,
+    labelHeight: number,
+    point: IPointData,
+) {
+    const { x, y } = getTooltipPositionInChartContainer.call(
+        this,
+        chartType,
+        stacking,
+        labelWidth,
+        labelHeight,
+        point,
+    );
+    const { top: containerTop, left: containerLeft } = this.chart.container.getBoundingClientRect();
+    const leftOffset = pageXOffset + containerLeft - HIGHCHARTS_TOOLTIP_TOP_LEFT_OFFSET;
+    const topOffset = pageYOffset + containerTop - HIGHCHARTS_TOOLTIP_TOP_LEFT_OFFSET;
+
+    return {
+        x: isTooltipShowedInFullScreen() ? leftOffset : leftOffset + x,
+        y: topOffset + y,
+    };
+}
+
+const isTooltipShowedInFullScreen = () => {
```

Comment:
One last minor typo which is my bad.
`isTooltipShowedInFullScreen` -> `isTooltipShownInFullScreen`

Reply:
updated